scripts/data_assimilation/da_no_data_mod.sh

```diff
@@ -17,6 +17,53 @@ else
   cycle=$2
```

Comment:
why do this in sh?   Wouldn't python be a better example?

Reply:
I guess it depends on what the DA community is doing. The idea is to test the DA cycle in a way that is similar to active DA experiments. Currently, all the DA scripts I have seen are shell scripts.

Reply:
> I'm not really sure of what distinction you are trying to make here and why the original implementation was inadequate.

Are you talking about da_no_data_mod.sh (the question is in a weird place).
The idea here is to check to see if DWAV noted and logged the resume signal.

Reply:
This question was in reference to the entire PR.  However since I am not the primary reviewer I can let this go.

Reply:
Ah. The reason for this PR is that we need separate mechanisms to support CESM2 data assimilation and post-CESM2 pause/resume functionality (including data assimilation).
To support current DA usage, we need to send a resume signal to the appropriate components at initialization and only at initialization. This PR does that using the `DATA_ASSIMILATION_XXX` variables. Prior to this PR, there was no resume signal going to the components at initialization time.

Pause/resume also needs to use the resume signal but at run time (and not at initialization time). I need the `PAUSE_ACTIVE_XXX` variables to determine which components will participate in pause (i.e., will receive pause signals). In pause/resume, the resume signal is not controlled by the configuration but by the active ESP component.

Pause/resume is not tied to data assimilation so I felt I needed separate variables.

Reply:
I confirm that all of the existing DART scripts use [[t]]c]sh.
I'm don't know enough about the CESM context to review
the PAUSE_ACTIVE_XXX changes, but it sounds like it's
still incorporating the functionality that DA needs, so I'll just
approve it.